src/Tribe/Rewrite.php

```diff
@@ -0,0 +1,166 @@
+<?php
+// Don't load directly
```

Comment:
Please avoid this, it makes loading these files in unit testing a difficult operation.